portfolio/src/main/java/com/google/sps/servlets/HomeServlet.java

```diff
@@ -0,0 +1,68 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.Gson;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/login")
+public class HomeServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html");
+
+    UserService userService = UserServiceFactory.getUserService();
+    if (userService.isUserLoggedIn()) {
+      boolean isLoggedIn = true;
+      String userEmail = userService.getCurrentUser().getEmail();
+      String urlToRedirectToAfterUserLogsOut = "/";
+      String logoutUrl = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
+
+      
+      response.setContentType("application/json;");
+      response.getWriter().println(convertToJson(isLoggedIn,logoutUrl));
+    } else {
+      boolean isLoggedIn = false;
+      String urlToRedirectToAfterUserLogsIn = "/";
+      String loginUrl = userService.createLoginURL(urlToRedirectToAfterUserLogsIn);
+      
+     response.setContentType("application/json;");
+     response.getWriter().println(convertToJson(isLoggedIn,loginUrl));
+
+      
+    }
+  }
+
+  //manually converts login status to boolean
+  private String convertToJson(boolean isLoggedIn, String logUrl) {
+    String json = "{";
+    json += "\"isLoggedIn\": ";
+    json += "\"" + Boolean.toString(isLoggedIn) + "\"";
```

Comment:
I think you can probably use `isLoggedIn` here directly, since it'll be converted to a `String` automatically?